src/Files.App/UserControls/SideBar/SideBarView.xaml

```diff
@@ -14,20 +14,21 @@
 	<Grid>
 		<Grid.ColumnDefinitions>
 			<ColumnDefinition
-				x:Name="DisplayColumn"
+				x:Name="PaneColumnDefinition"
 				Width="240"
 				MaxWidth="480" />
 			<ColumnDefinition Width="Auto" />
 			<ColumnDefinition Width="*" />
 		</Grid.ColumnDefinitions>
 
-		<!--  This is the background for the sidebar pane and sidebar content  -->
```

Comment:
Can you restore this comment? It's important to know that this affects the entire app.